core/loader_shared.c

```diff
@@ -412,6 +415,9 @@ privload_insert(privmod_t *after, app_pc base, size_t size, const char *name,
         mod->name = double_strrchr(mod->path, DIRSEP, ALT_DIRSEP);
         if (mod->name == NULL)
             mod->name = mod->path;
+        /* XXX: mod->name contains the leading '/'. We probably don't want that,
+         * but it doesn't seem to break anything, leaving it for now.
```

Comment:
Probably this path is never taken?

Reply:
It is taken.. I actually think that's a bug but didn't want to touch this as it didn't seem to break anything right now, hence the comment.

Reply:
P.S. you can see that at other call sites we increment the '/' away.

Reply:
The comment should reflect that then.  Probably the name field comment should reflect that.

Reply:
Added to comment.